pkgs/development/ocaml-modules/ppxfind/default.nix

```diff
@@ -12,14 +12,12 @@ buildDunePackage (rec {
 
 	buildInputs = [ ocaml-migrate-parsetree ];
 
+  dontStrip = true;
+
 	meta = {
 		homepage = "https://github.com/diml/ppxfind";
 		description = "ocamlfind ppx tool";
 		license = lib.licenses.bsd3;
 		maintainers = [ lib.maintainers.vbgl ];
 	};
-} // (
```

Comment:
is this change needed?

Reply:
I suppose not in light of your other suggestion